### ISSUE TITLE
Currently active menu is not conveyed programmatically

### ISSUE THREAD

**Issue by andrewhick** [alphagov/wcag-primer#132]
In the header, the currently active menu item, such as Overview or All Requirements, is conveyed visually (by changing the text from white to blue) but not programmatically.

While it meets [WCAG 1.4.1 Use of Colour](https://www.w3.org/WAI/WCAG22/Understanding/use-of-color) (because the contrast ratio between white and blue is at least 3:1) it fails [WCAG 1.3.1 Info and Relationships](https://www.w3.org/WAI/WCAG22/Understanding/info-and-relationships) because information is not conveyed programmatically - for example by using `aria-current`.

![Menu with items in white and All Requirements in blue](https://github.com/user-attachments/assets/3fa71131-b26f-4210-b9b1-82fc48630c4f)

Detected by @shabana-ali .

**Comment by andrewhick** [comment]
Discussed with the Tech Docs team and agreed to track this in their repository, but keeping it open here too for visibility for now: https://github.com/alphagov/tech-docs-gem/issues/403

**Comment by andrewhick** [comment]
That PR on the Tech Docs repo has been merged so it will hopefully appear in the next Tech Docs release.
